test/models/album_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class AlbumTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+  test "A Album must have a title" do
+    little_women = Album.new
+    little_women.valid? "An Album without a title is invalid."
```

Comment:
Like the later tests, this test should set the other fields on the model before checking its validity (otherwise we are not ensuring that it's _only_ because of the title that our validation is failing).